lib/xray.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const AWSXRay = require('aws-xray-sdk');
+const AWS = require('aws-sdk');
+
+const fake = {
+  addAnnotation: Function.prototype,
+};
+
+const asyncFunctionWrapper = (context, fn, resolve, reject) => async (subsegment) => {
+  try {
+    const result = await fn({
+      addAnnotation(key, value) {
+        try {
+          subsegment.addAnnotation(key, value);
+        } catch (err) {
+          context.log.warn({ err }, 'Unable to addAnnotation to x-ray subsegment');
```

Comment:
The driver should take log `level` and use `log-me-gently` for logging https://github.com/alpha-lambda/aws-drivers/blob/master/lib/driver.js#L20